# PID controller gains

float32 p
float32 i
float32 d
float32 feedback_min_offset
float32 feedback_multiplier
float32 feedforward_min_offset
float32 feedforward_fraction
